web/src/app/lists/PaginatedList.js

```diff
@@ -170,27 +184,38 @@ export class PaginatedList extends React.PureComponent {
 
     // provide a message to display if there are no results
     emptyMessage: p.string,
+
+    // renders as an infinite list instead of the default
+    // page by page behavior
+    infiniteScroll: p.bool,
+
+    // to be used if using the infinite scroller
+    hasMore: p.bool,
```

Comment:
We can determine if there are more to load by checking if `loadMore` was provided, e.g. `Boolean(props.loadMore)`